src/main/kotlin/blackjack/domain/Cards.kt

```diff
@@ -0,0 +1,24 @@
+package blackjack.domain
+
+class Cards {
+    private val _group: MutableList<Card> = mutableListOf()
+    val group: List<Card> get() = _group.toList()
```

Comment:
아래와 같이 생성자 파라미터를 활용하여 초기화하면, 테스트 코드 시 함수를 통하지 않더라도, 사전 데이터 주입을 더 손쉽게 할 수 있습니다.

```suggestion
class Cards(cardGroup: List<Card>) {
    private val _group: MutableList<Card> = cardGroup.toMutableList()
    val group: List<Card> get() = _group.toList()
```

테스트 코드 작성만을 위해서 생긴 코드라고 생각한다면 맘에 안 드는 변경사항일 수 있는데요, 석호님은 어떻게 생각하세요?

Reply:
저는 사실 테스트코드만을 위한 코드 변경은 해본적이 없어서 그런지 굳이 필요할까라고 생각이 들기는 합니다!
혹시 수현님은 이렇게 변경했을때 경험하신 이점들이 따로 있으셨을까요?